honeybee_radiance/properties/model.py

```diff
@@ -392,6 +570,29 @@ def _check_and_add_orphaned_shade_modifier(self, obj, modifiers):
             if not self._instance_in_array(generic_context, modifiers):
                 modifiers.append(generic_context)
 
+    @staticmethod
+    def _sort_shades(shades):
```

Comment:
It is semantics but to me this is more of a separate or get shades by opaque more then sort.